src/Microsoft.AspNetCore.JsonPatch/Helpers/ConversionHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNetCore.JsonPatch.Helpers
```

Comment:
There are existing classes which are internal and use this namespace. I left it as it is currently so that we can clean up all these in a separate task.